Sage/carbon
# chore(pages): remove classic theme

Removal of classic theme from Pages component. Delete redundant documentation. Refactor storybook
example from storiesOf to CSF and provide an MDX example of the Pages component.

BREAKING CHANGE: Removal of classic theme from the Pages component. Component no longer supports
classic theme.

fixes FE-3117

### Proposed behaviour
- Component should not support `classic` theme
- Documentation is now included in an MDX example
- Stories file is now in the CSF

### Current behaviour
- Component supports `classic` theme
- Documentation complies with our previous standards
- Stories file is in the depreciated storiesOf format

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout FE-3117-pages_remove_classic_theme`
- `npm start` to start Storybook
- Navigate to the Pages story
- No classic theme example should be present
- Documentation should be in MDX format
